boards/fanke/fk7b0m1_vbt6/fk7b0m1_vbt6.dts

```diff
@@ -75,6 +76,61 @@
 	d3ppre = <2>;
 };
 
+&octospi1 {
+	pinctrl-0 = <&octospim_p1_clk_pb2 &octospim_p1_ncs_pb6
+		     &octospim_p1_io0_pd11 &octospim_p1_io1_pd12
+		     &octospim_p1_io2_pe2 &octospim_p1_io3_pd13>;
+	pinctrl-names = "default";
+	status = "okay";
+
+	/* Winbond external flash */
+	w25q64jvssiq_qspi: qspi-nor-flash@0 {
+		compatible = "st,stm32-ospi-nor";
+		reg = <0>;
+		ospi-max-frequency = <DT_FREQ_M(133)>;
+		size = <DT_SIZE_M(64)>; /* 64 Mbits */
```

Comment:
`size`  is now deprecated, see https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/migration-guide-3.6.rst#flash

Reply:
Done. Thanks!